src/components/Layouts/Footer/Partials/constants.js

```diff
@@ -0,0 +1,31 @@
+const newsList = [
```

Comment:
As this `constants.js` is a generic constants file, in the future we might need another constant, so newsList should not become the default export as it's not the main reason/focus of the file 👍 .